Writerside/topics/globalEvents_flags.md

```diff
@@ -0,0 +1,71 @@
+# Global Events Flags
+
+## dictionary
+This json contains a dictionary of key: value pairs, representing the globalEvent flag (which correlates to its type) returned from the Helldivers 2 Status API endpoint.
+
+```json
```

Comment:
I would avoid copying the contents of `globalEvents_flags` here as any update to that file would also need to be made here.
Preferably link to the file itself to make maintenance easier